SU2_CFD/include/numerics/NEMO/convection/lax.hpp

```diff
@@ -37,14 +37,17 @@
  */
 class CCentLax_NEMO : public CNEMONumerics {
 private:
-  unsigned short iDim, iVar; /*!< \brief Iteration on dimension and variables. */
+  unsigned short iDim, iVar, jVar; /*!< \brief Iteration on dimension and variables. */
```

Comment:
You should avoid keeping this type of variable here. Forces the compiler to write them back instead of making them throw-away local variables.